pr-template.md

```diff
@@ -1,35 +1,30 @@
 #### What does this PR do? (please provide any background)
 
-
 #### What tests does this PR have?
 
-
 #### How can this be tested?
 
+#### What domains are affected?
 
-#### Screenshots / Screencast
+#### Any tech debt?
 
+#### Screenshots / Screencast
 
 #### What gif best describes how you feel about this work?
-![](url)
-
----
+![]()
 
 - [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.
 
-#### Reviewers
-
-**Review 1**
-- [ ] :+1:
-
-**Review 2**
-- [ ] :+1:
+---
 
-**Review 3**
+**Reviewer**
 - [ ] :+1:
+- [ ] This PR need any additional reviewers!
```

Comment:
Can we rephrase this to the negative. "I don't think this PR needs and additional reviewers"

This way indicating that it does need reviewers doesn't move the progress bar in the PR list view along.